projects.go

```diff
@@ -1270,6 +1270,8 @@ type HookCustomHeader struct {
 type ProjectHook struct {
 	ID                        int                 `json:"id"`
 	URL                       string              `json:"url"`
+	Name                      string              `json:"name,omitempty"`
+	Description               string              `json:"description,omitempty"`
```

Comment:
You don't need the `omitempty` on the main struct used for reads, if you wouldn't mind removing it.

Reply:
Fixed!